wp-parsely.php

```diff
@@ -21,6 +21,189 @@
 	'3.1',
 ];
 
+/**
+ * Class to hold Parse.ly loading info
+ * Will prevent this information from being queried multiple times
+ * by keeping the status in a static variable
+ */
+final class Parsely_Loader_Info {
+
```

Comment:
This empty line (30) can be removed.

Reply:
Great! I have removed this line in the latest commit.